RELEASE.md

```diff
@@ -0,0 +1,19 @@
+# Releasing the SDK
```

Comment:
❓ Should we mention that we follow SemVer and [link](https://semver.org/) to it?